### DISCUSSION TITLE
Cannot read properties of undefined (reading 'init')

### DISCUSSION THREAD

**Discussion by siosio34** [WalletConnect/walletconnect-monorepo#2995] [Developer Support]
When I refresh the page, I get the following error. Could you please let me know the solution for this issue?

wagmi - version 1.3.2
walletconnect-version 2.8.4

<img width="417" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/7166022/5193649d-5412-4f7f-97e6-8d003bbcb6b6">
<img width="417" alt="image" src="https://github.com/WalletConnect/walletconnect-monorepo/assets/7166022/10658f0a-7c52-4efb-91ab-790c49765999">

**Comment by emzet93** [comment]
Try to delete `node_modules` directory and reinstall dependencies. It helped in my case, I guess it's maybe some versions conflict in node modules or sth.

**Reply by siosio34** [reply]
thanks @emzet93
